tests/pfc/files/configs/pfc.py

```diff
@@ -0,0 +1,399 @@
+                duration=Duration(Continuous(delay=0, delay_unit='nanoseconds'))
+            )
+        elif (pause_frame_type == 'global') :
+            pause = Header(EthernetPause(
+                dst=FieldPattern(choice='01:80:C2:00:00:01'),
+                src=FieldPattern(choice='00:00:fa:ce:fa:ce')
+            ))
+
+            pause_flow = Flow(
+                name='Pause Storm',
+                tx_rx=TxRx(pause_endpoint),
+                packet=[pause],
+                size=Size(64),
+                rate=Rate('line', value=pause_line_rate),
+                duration=Duration(Continuous(delay=0, delay_unit='nanoseconds'))
+            )
+        else :
+            pass   
+
+        config.flows.append(pause_flow)
+
+    return one_hundred_gbe
+
+
+@pytest.fixture
+def start_delay(request):
+    return request
+
+
+@pytest.fixture
+def traffic_duration(request):
+    return request
+
+
+@pytest.fixture
+def bw_multiplier(request):
+    return request
+
+
+@pytest.fixture
+def pause_line_rate(request):
+    return request
+
+
+@pytest.fixture
+def traffic_line_rate(request):
+    return request
+
+
+@pytest.fixture
+def frame_size(request):
+    return request
+
+
+@pytest.fixture(scope='session')
+def serializer(request):
+    class Serializer(object):
+        def __init__(self, request):
+            self.request = request
+            self.test_name = getattr(request.node, "name")
+
+        def json(self, obj):
+            import json
+            json_str = json.dumps(obj, indent=2, default=lambda x: x.__dict__)
+            return '\n[%s] %s: %s\n' % (self.test_name, obj.__class__.__name__, json_str)
+
+        def yaml(self, obj):
+            import yaml
+            yaml_str = yaml.dump(obj, indent=2)
+            return '\n[%s] %s: %s\n' % (self.test_name, obj.__class__.__name__, yaml_str)
+
+        def obj(self, json_string):
+            return json.loads(json_string, object_hook=self._object_hook)
+
+        def _object_hook(self, converted_dict):
+            return namedtuple('X', converted_dict.keys())(*converted_dict.values())
+
+    return Serializer(request)
+
+
+@pytest.fixture
+def port_bandwidth(conn_graph_facts,
+                   fanout_graph_facts,
+                   bw_multiplier) :
+
+   fanout_devices = IxiaFanoutManager(fanout_graph_facts)
+   fanout_devices.get_fanout_device_details(device_number=0)
+   device_conn = conn_graph_facts['device_conn']
+   available_phy_port = fanout_devices.get_ports()
+   reference_peer = available_phy_port[0]['peer_port']
+   reference_speed = int(device_conn[reference_peer]['speed'])
+
+   for intf in available_phy_port:
+        peer_port = intf['peer_port']
+        intf['speed'] = int(device_conn[peer_port]['speed'])
+        pytest_assert(intf['speed'] == reference_speed,
+            "speed of all the ports are not same")
+
+   return reference_speed * bw_multiplier
+
+
+@pytest.fixture
+def one_hundred_gbe(conn_graph_facts,
```

Comment:
The name of 'one_hundred_gbe' is not so straightforward. How about changing the name to 'l1_config' and extending this function to support any link speeds.

Reply:
The name 'one_hundred_gbe' has been replaced by the name 'l1_config' everywhere in the code.